src/components/combobox/combobox.tsx

```diff
@@ -96,19 +96,29 @@ export class Combobox
   @Prop({ reflect: true, mutable: true }) active = false;
 
   @Watch("active")
-  @Watch("open")
-  activeHandler(): void {
+  activeHandler(value: boolean): void {
     if (this.disabled) {
```

Comment:
I think you can get rid of all the logic here and just have the handler set the open value.